tests/sentry/mediators/external_requests/test_alert_rule_action_requester.py

```diff
@@ -0,0 +1,71 @@
+import responses
+
+from sentry.mediators.external_requests import AlertRuleActionRequester
+from sentry.testutils import TestCase
+from sentry.utils import json
+from sentry.utils.sentryappwebhookrequests import SentryAppWebhookRequestsBuffer
+
+
+class TestAlertRuleActionRequester(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = self.create_user(name="foo")
+        self.org = self.create_organization(owner=self.user)
+        self.project = self.create_project(slug="boop", organization=self.org)
+        self.group = self.create_group(project=self.project)
+
+        self.sentry_app = self.create_sentry_app(
+            name="foo", organization=self.org, webhook_url="https://example.com", scopes=()
+        )
+
+        self.install = self.create_sentry_app_installation(
+            slug="foo", organization=self.org, user=self.user
+        )
+
+    @responses.activate
+    def test_makes_request(self):
+        fields = {"title": "An Alert", "description": "threshold reached", "assignee": "user-1"}
+
+        responses.add(
+            method=responses.POST,
+            url="https://example.com/alert-rule",
+            json={
```

Comment:
@AniketDas-Tekky hmm for now, we don't expect any json back. We only expect a 200 from the provider.